### DISCUSSION TITLE
Change #page height from 100% to auto/unset

### DISCUSSION THREAD

**Discussion by Axos11** [bootscore/bootscore#391] [General]
I ran into some issues with screen recording tools (mouseflow) if the header#masthead had 0 height. 
Is there any reason for keeping it that way that I am missing?

I also got some vertical margin addition between the breadcrumbs and the content if I don't set `#page { display: block }`. The bottom margin of the breadcrumbs and the top content margin seem to be not collapsing if `#page { display: flex}`

**Comment by crftwrk** [comment]
> I ran into some issues with screen recording tools (mouseflow) if the header#masthead had 0 height.

Why should `#masthead` have a height? This is just the wrapper for the `navbar`.

`#page` has `display: flex` for the sticky footer. Check here  https://dev.bootscore.me/theme/sticky-footer/.

**Reply by Axos11** [reply]
Because it seems that screenrecordings like mouseflow get the mousepointer wrong. Because the only child in #masthead is sticky, #masthead gets a height of 0 and that seems to confuse the y offset of recordings. Sadly I have no recording from before the fix.

"There is only an empty page at that link. How can I visualize the sticky footer?"
EDIT: 
I got your point now. Don't have a solution for now that would solve both "needs". Maybe someone else has some idea for some "CSS Magic" how to get the masthead some height without ruining the sticky footer.

**Reply by crftwrk** [reply]
If you have pages with enough content that does not place the footer in the middle of the screen, why not simply override `#page {display: block;}`? The sticky footer is just for edge cases like a tiny imprint page on large screens.

**Reply by Axos11** [reply]
I did that already on most of my pages and I do know, that if its an edge case that only happens for me, we shouldn't change the core.
I was just looking for a solution that fixes things on both ends :)
